hw03_millere4.py

```diff
@@ -0,0 +1,48 @@
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
```

Comment:
Remove any code and comments that are unnecessary. These are leftover from my template file.